bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -69,6 +69,63 @@ spec:
             cpu: 100m
             memory: 300Mi
 
+      # ovn-nbctl daemon 
+      - name: ovn-nbctl
+        image: {{.OvnImage}}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          if [[ -f /env/_master ]]; then
+            set -o allexport
+            source /env/_master
+            set +o allexport
+          fi
+          ovn_config_namespace=openshift-ovn-kubernetes
+          retries=0
+          while true; do
+            db_ip=$(kubectl get ep -n ${ovn_config_namespace} ovnkube-db -o jsonpath='{.subsets[0].addresses[0].ip}')
+            if [[ -n "${db_ip}" ]]; then
+              break
+            fi
+            (( retries += 1 ))
+            if [[ "${retries}" -gt 40 ]]; then
+              echo "db endpoint never came up"
+              exit 1
+            fi
+            echo "waiting for db endpoint"
+            sleep 5
+          done
+          echo "db_ip ${db_ip}"
+          rm -f /var/run/openvswitch/ovn-nbctl.pid
+          rm -f /var/run/openvswitch/ovn-nbctl.*.ctl
+          exec ovn-nbctl --detach --pidfile --no-chdir \
+            --db="tcp:${db_ip}:{{.OVN_NB_PORT}}" \
+            "-vconsole:${OVN_LOG_LEVEL}" --log-file="/var/log/openvswitch/ovn-nbctl.log"
+          sleep 1
```

Comment:
you do an `exec` - nothing after it in the shell script will be executed.

Reply:
OK